tests/functional/openstackdataplanedeployment_controller_test.go

```diff
@@ -186,4 +200,230 @@ var _ = Describe("Dataplane Deployment Test", func() {
+				g.Expect(th.K8sClient.Status().Update(th.Ctx, &baremetalAlpha)).To(Succeed())
+				// OpenStackBaremetalSet has the same name as OpenStackDataPlaneNodeSet
+				g.Expect(th.K8sClient.Get(th.Ctx, betaNodeSetName, &baremetalBeta)).To(Succeed())
+				baremetalBeta.Status.Conditions.MarkTrue(
+					baremetalv1.OpenStackBaremetalSetBmhProvisioningReadyCondition,
+					baremetalv1.OpenStackBaremetalSetBmhProvisioningReadyMessage)
+				g.Expect(th.K8sClient.Status().Update(th.Ctx, &baremetalBeta)).To(Succeed())
+
+			}, th.Timeout, th.Interval).Should(Succeed())
+
+			// Create all services necessary for deployment
+			for _, serviceName := range nodeSetAlpha.Spec.Services {
+				dataplaneServiceName := types.NamespacedName{
+					Name:      serviceName,
+					Namespace: namespace,
+				}
+				service := GetService(dataplaneServiceName)
+				executionName := dataplaneutil.GetAnsibleExecutionNamePrefix(service)
+
+				//Retrieve service AnsibleEE and set JobStatus to Successful
+				Eventually(func(g Gomega) {
+					// Make an AnsibleEE name for each service
+					ansibleeeName := types.NamespacedName{
+						Name:      fmt.Sprintf("%s-%s", executionName, dataplaneMultiNodesetDeploymentName.Name),
+						Namespace: dataplaneMultiNodesetDeploymentName.Namespace,
+					}
+					ansibleEE := &ansibleeev1.OpenStackAnsibleEE{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      ansibleeeName.Name,
+							Namespace: ansibleeeName.Namespace,
+						}}
+					g.Expect(th.K8sClient.Get(th.Ctx, ansibleeeName, ansibleEE)).To(Succeed())
+					ansibleEE.Status.JobStatus = ansibleeev1.JobStatusSucceeded
+
+					g.Expect(th.K8sClient.Status().Update(th.Ctx, ansibleEE)).To(Succeed())
+				}, th.Timeout, th.Interval).Should(Succeed())
+			}
+
+			// Create all services necessary for deployment
+			for _, serviceName := range nodeSetBeta.Spec.Services {
+				dataplaneServiceName := types.NamespacedName{
+					Name:      serviceName,
+					Namespace: namespace,
+				}
+				service := GetService(dataplaneServiceName)
+				executionName := dataplaneutil.GetAnsibleExecutionNamePrefix(service)
+				//Retrieve service AnsibleEE and set JobStatus to Successful
+				Eventually(func(g Gomega) {
+					// Make an AnsibleEE name for each service
+					ansibleeeName := types.NamespacedName{
+						Name:      fmt.Sprintf("%s-%s", executionName, dataplaneMultiNodesetDeploymentName.Name),
+						Namespace: dataplaneMultiNodesetDeploymentName.Namespace,
+					}
+					ansibleEE := &ansibleeev1.OpenStackAnsibleEE{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      ansibleeeName.Name,
+							Namespace: ansibleeeName.Namespace,
+						}}
+					g.Expect(th.K8sClient.Get(th.Ctx, ansibleeeName, ansibleEE)).To(Succeed())
+					ansibleEE.Status.JobStatus = ansibleeev1.JobStatusSucceeded
+
+					g.Expect(th.K8sClient.Status().Update(th.Ctx, ansibleEE)).To(Succeed())
+				}, th.Timeout, th.Interval).Should(Succeed())
+			}
```

Comment:
could you assert that the AEE for the global service has the host override variable set to all while the rest of the services has that variable set to the nodeset name? I'm OK to do that in a follow up